qa/standalone/scrub/osd-scrub-test.sh

```diff
@@ -599,14 +599,14 @@ function TEST_dump_scrub_schedule() {
     ceph pg $pgid scrub
     sleep 1
     sched_data=()
-    declare -A expct_scrub_peri_sched=( ['query_is_future']="false" )
-    wait_any_cond $pgid 10 $saved_last_stamp expct_scrub_peri_sched "waitingBeingScheduled" sched_data || return 1
+    declare -A expect_scrub_peri_sched=( ['query_is_future']="false" )
+    wait_any_cond $pgid 10 $saved_last_stamp expect_scrub_peri_sched "waitingBeingScheduled" sched_data || return 1
```

Comment:
Leave this alone, shortening in code is not uncommon, we gain nothing by changing it.